javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -246,15 +246,17 @@ export type InstalledPluginListItem = {
   version: string;
 };
 
+export type DaoMetadata = {
```

Comment:
Didn't we already have such a type before?

Reply:
Yes, I didn't saw it